src/pages/workspace/WorkspaceJoinUserPage.tsx

```diff
@@ -39,7 +39,7 @@ function WorkspaceJoinUserPage({route, policy}: WorkspaceJoinUserPageProps) {
     }, []);
 
     useEffect(() => {
-        if (!policy || isUnmounted.current || isJoinLinkUsed) {
+        if (isUnmounted.current || isJoinLinkUsed) {
```

Comment:
@nkdengineer  Why do you remove this condition `!policy` ? Should we check `isEmptyObject(policy)` instead of removing it

Reply:
@DylanDylann  In the past we use `getPolicy` function which returns empty object by default so the check `!policy` is always `true`. Now, we're get `policy` from Onyx so we should remove this check that is preventing join workspace flow. 

`isEmptyObject(policy)` also returns true for `undefined` or `null`.